source/features/releases-tab.tsx

```diff
@@ -52,31 +60,58 @@ async function init(): Promise<false | void> {
 	}
 
 	const releasesTab = (
-		<a href={`/${repoUrl}/releases`} className="reponav-item" data-hotkey="g r">
+		<a href={`/${repoUrl}/releases`} className="reponav-item" data-hotkey="g r" data-tab-item="releases-tab">
 			<TagIcon/>
-			<span> Releases </span>
+			<span data-content="Releases"> Releases </span>
 			{count === undefined ? '' : <span className="Counter">{count}</span>}
 		</a>
 	);
 
 	await elementReady('.pagehead + *'); // Wait for the tab bar to be loaded
+
+	const repoNavigationBar = select('.js-repo-nav')!;
+
+	if (repoNavigationBar.classList.contains('UnderlineNav')) {
+		// Adapt releases tab to GitHub's repository redesign
+		releasesTab.classList.remove('reponav-item');
+		releasesTab.classList.add(
+			'js-selected-navigation-item',
+			'UnderlineNav-item',
+			'hx_underlinenav-item',
+			'no-wrap',
+			'js-responsive-underlinenav-item'
+		);
+		select('svg', releasesTab)!.classList.add('UnderlineNav-octicon');
+	}
+
 	appendBefore(
-		// GHE doesn't have `reponav > ul`
-		select('.reponav > ul') ?? select('.reponav')!,
+		// GHE doesn't have `.js-repo-nav > ul`
+		select(':scope > ul', repoNavigationBar) ?? repoNavigationBar,
```

Comment:
Are you sure `GHE` currently has `js-repo-nav`?

Reply:
No, I am not.

@busches I think you have access to a GHE instance, right? Can you confirm this?

Reply:
Probably isn't necessary since the suggested solution is to completely fork the code, so this section would remain untouched.